pkg/config/evaluation.go

```diff
@@ -161,6 +164,46 @@ func (s *StepCondition) evaluateV1(config StepConfig, utils piperutils.FileUtils
 		return checkForNpmScriptsInPackagesV1(s.NpmScript, config, utils)
 	}
 
+	// TODO: complex CPE value checks : currently implemented for boolean and string cpe values directly from the file and not for deep json inside the cpe file
+	// only the first condition will be evaluated.
+	// if multiple conditions should be checked they need to provided via the Conditions / NotActiveConditions list
+	if s.Cpe != nil {
+		if len(s.Cpe) > 1 {
+			return false, errors.Errorf("only one cpe key allowed per condition but %v provided", len(s.Cpe))
+		}
+
+		cpeBasePath := filepath.Join(".pipeline", "commonPipelineEnvironment")
```

Comment:
I guess it is better to use the original logic here:
https://github.com/SAP/jenkins-library/blob/master/pkg/config/stepmeta.go#L362

We should not duplicate knowledge about path, file types and param resolution here.

Usage as per https://github.com/SAP/jenkins-library/blob/master/cmd/piper.go#L320

With that it should be possible to just refer to the cpe as it is done in the step metadata.


With that